2020-04-25 16:42:10,150 ERROR: Exception on /reset_password_request [POST] [in /Users/andrew.xia/Documents/Github/flasky/venv/lib/python3.6/site-packages/flask/app.py:1891]
Traceback (most recent call last):
  File "/Users/andrew.xia/Documents/Github/flasky/venv/lib/python3.6/site-packages/flask/app.py", line 2446, in wsgi_app
    response = self.full_dispatch_request()
  File "/Users/andrew.xia/Documents/Github/flasky/venv/lib/python3.6/site-packages/flask/app.py", line 1951, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/Users/andrew.xia/Documents/Github/flasky/venv/lib/python3.6/site-packages/flask/app.py", line 1820, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/Users/andrew.xia/Documents/Github/flasky/venv/lib/python3.6/site-packages/flask/_compat.py", line 39, in reraise
    raise value
  File "/Users/andrew.xia/Documents/Github/flasky/venv/lib/python3.6/site-packages/flask/app.py", line 1949, in full_dispatch_request
    rv = self.dispatch_request()
  File "/Users/andrew.xia/Documents/Github/flasky/venv/lib/python3.6/site-packages/flask/app.py", line 1935, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/Users/andrew.xia/Documents/Github/flasky/app/routes.py", line 166, in reset_password_request
    send_password_reset_email(user)
  File "/Users/andrew.xia/Documents/Github/flasky/app/email.py", line 13, in send_password_reset_email
    sender=app.config['ADMINS'][0],
NameError: name 'app' is not defined
2020-04-25 16:42:36,190 ERROR: Exception on /reset_password_request [POST] [in /Users/andrew.xia/Documents/Github/flasky/venv/lib/python3.6/site-packages/flask/app.py:1891]
Traceback (most recent call last):
  File "/Users/andrew.xia/Documents/Github/flasky/venv/lib/python3.6/site-packages/flask/app.py", line 2446, in wsgi_app
    response = self.full_dispatch_request()
  File "/Users/andrew.xia/Documents/Github/flasky/venv/lib/python3.6/site-packages/flask/app.py", line 1951, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/Users/andrew.xia/Documents/Github/flasky/venv/lib/python3.6/site-packages/flask/app.py", line 1820, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/Users/andrew.xia/Documents/Github/flasky/venv/lib/python3.6/site-packages/flask/_compat.py", line 39, in reraise
    raise value
  File "/Users/andrew.xia/Documents/Github/flasky/venv/lib/python3.6/site-packages/flask/app.py", line 1949, in full_dispatch_request
    rv = self.dispatch_request()
  File "/Users/andrew.xia/Documents/Github/flasky/venv/lib/python3.6/site-packages/flask/app.py", line 1935, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/Users/andrew.xia/Documents/Github/flasky/app/routes.py", line 166, in reset_password_request
    send_password_reset_email(user)
  File "/Users/andrew.xia/Documents/Github/flasky/app/email.py", line 13, in send_password_reset_email
    sender=app.config['ADMINS'][0],
NameError: name 'app' is not defined
2020-04-25 16:43:09,013 INFO: Microblog startup [in /Users/andrew.xia/Documents/Github/flasky/app/__init__.py:48]
2020-04-25 16:48:44,302 INFO: Microblog startup [in /Users/andrew.xia/Documents/Github/flasky/app/__init__.py:48]
2020-04-25 21:28:26,854 INFO: Microblog startup [in /Users/andrew.xia/Documents/Github/flasky/app/__init__.py:50]
2020-04-25 21:29:37,201 INFO: Microblog startup [in /Users/andrew.xia/Documents/Github/flasky/app/__init__.py:50]
2020-04-25 21:31:29,008 INFO: Microblog startup [in /Users/andrew.xia/Documents/Github/flasky/app/__init__.py:50]
2020-04-25 21:41:55,139 INFO: Microblog startup [in /Users/andrew.xia/Documents/Github/flasky/app/__init__.py:52]
2020-04-25 21:45:27,756 INFO: Microblog startup [in /Users/andrew.xia/Documents/Github/flasky/app/__init__.py:52]
2020-04-26 16:20:22,879 INFO: Microblog startup [in /Users/andrew.xia/Documents/Github/flasky/app/__init__.py:58]
2020-04-26 16:20:53,124 INFO: Microblog startup [in /Users/andrew.xia/Documents/Github/flasky/app/__init__.py:58]
2020-04-26 16:22:03,464 INFO: Microblog startup [in /Users/andrew.xia/Documents/Github/flasky/app/__init__.py:58]
2020-04-26 16:24:57,308 INFO: Microblog startup [in /Users/andrew.xia/Documents/Github/flasky/app/__init__.py:59]
2020-04-26 16:48:49,958 INFO: Microblog startup [in /Users/andrew.xia/Documents/Github/flasky/app/__init__.py:59]
2020-04-26 16:49:07,596 INFO: Microblog startup [in /Users/andrew.xia/Documents/Github/flasky/app/__init__.py:59]
2020-04-26 17:02:14,226 INFO: Microblog startup [in /Users/andrew.xia/Documents/Github/flasky/app/__init__.py:59]
